tests/utils.py

```diff
@@ -198,3 +200,16 @@ def create_iamc_query_test_data(test_mp):
     run2.meta = {"run": 2, "test": "string", "bool": False}
 
     return [r1, r2, r3], units
+
+
+def create_indexsets_for_run(
+    platform: Platform, run_id: int, amount: int = 2, offset: int = 0
```

Comment:
You could set offset to 1 as default and remove the "+1" on line 212.

Reply:
Good catch, I did not work with full concentration, it seems :sweat_smile: